packages/components/src/blob/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useRef, useEffect, useCallback } from "react";
+import { canvasPath } from "blobs/v2/";
+import { colors } from "@variant/profile";
+
+type VariantBlob = {
+  seed?: string;
+  extraPoints?: number;
+  randomness?: number;
+  useAnimation?: boolean;
+  animationSpeed?: number;
+  width: number;
+  height: number;
+  imgSource?: string;
+  color?: colors.ValidDefaultColor;
+} & JSX.IntrinsicElements["canvas"];
+
+export const VariantBlob: React.FC<VariantBlob> = React.memo(
+  ({
+    width,
+    height,
+    seed = Math.random(),
+    extraPoints = 4,
+    randomness = 9,
+    color = colors.colorPairs.primary.default.bg.toString(),
+    imgSource,
+    ...rest
+  }) => {
+    const canvasRef = useRef<HTMLCanvasElement>(null);
+
+    const generateBlob = useCallback(
+      (ctx: CanvasRenderingContext2D, image?: HTMLImageElement) => {
+        // Could store in session for more persistent "randomness" on blobstyle
+        const blob = canvasPath({
+          seed,
+          extraPoints,
+          randomness,
+          size: Math.min(width, height),
+        });
+        if (image) {
+          ctx.clip(blob);
+          ctx.drawImage(image, 0, 0, width, height);
+        } else {
+          ctx.fillStyle = color;
+          ctx?.fill(blob);
+        }
+      },
+      []
```

Comment:
I think we need to add `seed`, `extraPoints`, `randomness`, `width`, `height`, `image` and `blob` as dependencies here.

Reply:
Fixed this, but done after moving file, so changes isn't reflected propperly